lib/connect-session-sequelize.js

```diff
@@ -25,16 +29,29 @@ module.exports = function SequelizeSessionInit(Store) {
 		}
 		Store.call(this, options);
 
-    		// Check if specific table should be used for DB connection
-    		if (options.table) {
-      			debug('Using table: %s for sessions', options.table);
-      			// Get Specifed Table from Sequelize Object
-      			this.sessionModel =  options.db[options.table] || options.db.models[options.table];
-    		} else {
-    			// No Table specified, default to ./model
-      			debug('No table specified, using default table.');
-      			this.sessionModel = options.db.import(path.join(__dirname, 'model'));
-    		}
+		var key;
+		this.options = {};
+		for (key in defaultOptions) {
+			this.options[key] = defaultOptions[key];
```

Comment:
My only hesitation with assigning `defaultOptions` directly to `this.options` is that if `defaultOptions` was ever modified, it would affect `this.options`. Since we don't expose `defaultOptions` anywhere this is a pretty unlikely case so I will go ahead and change it. I guess just be wary of future changes modifying `defaultOptions`; although I'm not really sure why they would  ¯\_(ツ)_/¯.